cassandra/writer.go

```diff
@@ -147,11 +152,14 @@ func (w *Writer) worker() {
 			err := w.write(id, s.TimestampMS, s.Value)
 			if err != nil {
 				// send error back on payload's errch; don't block the worker
+				go ll.WithError(err).Error("write failed")
```

Comment:
Let's not log the error right here. Let's just pass back the error and then the component that created the cassandra writer can decide what to do with the error (including whether or not to log it).